addon/instance-initializers/current-when.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+const { EmberENV } = config;
+
+export function initialize(appInstance) {
+  if (EmberENV._ENABLE_CURRENT_WHEN_SUPPORT !== true) {
+    return false;
+  }
+
+  const linkToFactory = appInstance.factoryFor('component:link-to');
+
+  linkToFactory.class.reopen({
```

Comment:
Instead of an initializer can we do this directly in a vendor file + app.import?  My main concerns here are: 1) initializers are ran for every acceptance test and we don't want to do this many times and 2) I'm not sure how engines would inherit this initializer from their host apps.

Reply:
Ya thats a good point about engines